authzed/api/v1/md.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package authzed.api.v1;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v1";
+option java_package = "com.authzed.api.v1";
+
+extend google.protobuf.EnumValueOptions {
+  optional string string_name = 123456789;
```

Comment:
I know this doesn't have to be this number, but it's what was in the proto docs and I couldn't track down any docs on what values make sense here (I'm sure they exist).